nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -90,4 +132,141 @@ private static void invalidInstantiationError(
         errorBuilder.createErrorDescription(
             errorMessage, analysis.buildDescription(tree), state, null));
   }
+
+  private static void invalidAssignmentInstantiationError(
+      Tree tree,
+      String lhsAnnotation,
+      String rhsAnnotation,
+      VisitorState state,
+      NullAway analysis) {
+    ErrorBuilder errorBuilder = analysis.getErrorBuilder();
+    ErrorMessage errorMessage =
+        new ErrorMessage(
+            ErrorMessage.MessageTypes.TYPE_PARAMETER_CANNOT_BE_NULLABLE,
+            String.format(
```

Comment:
This message isn't quite right.  Maybe we can pass in the full (original) lhs and rhs types, and then write "Cannot assign from type " + rhsType + " to type " + lhsType + " due to mismatched nullability of type parameters"